StablecoinResearch/Stablecoins/DataInsights/raw_data/fetch_ust_prices.py

```
@@ -0,0 +1,39 @@
+# Convert to UNIX timestamps (milliseconds)
+start_ts = int(start_dt.timestamp())
+end_ts = int(end_dt.timestamp())
+
+params = {
+    'vs_currency': 'usd',
+    'from': start_ts,
+    'to': end_ts
+}
+
+print(f"Fetching UST price data from {start_dt} to {end_dt}...")
+resp = requests.get(COINGECKO_API, params=params)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)